sass/routes/_settings.scss

```diff
@@ -374,11 +390,11 @@
                     .theme-elements__body {
                         @include border-radius(0 0 3px 3px);
                         @include legacy-pie-clearfix;
-                        @include alpha-property(background-color, $white, 0.05);
 
```

Comment:
Do we need to keep using these mixins? Unless we need some type of calculation I believe we can replace all of them with CSS props

Reply:
good question ... IMO this is certainly possible, but out of scope for this PR (its big enough as it is already)

Reply:
Can we create tickets for those? Or are there already?